src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/CaseloadItem.kt

```diff
@@ -13,9 +14,11 @@ data class CvlFields(
   val licenceType: LicenceType,
 
   @Schema(description = "Date which the hard stop period will start", example = "2023-05-01")
+  @JsonFormat(pattern = "dd/MM/yyyy")
   var hardStopDate: LocalDate? = null,
 
   @Schema(description = "Date which to show the hard stop warning", example = "2023-05-01")
+  @JsonFormat(pattern = "dd/MM/yyyy")
```

Comment:
Should the example match the pattern format?

Reply:
yes, it should! - thanks will fix in follow up